### PR TITLE
pam_faillock:

### PR DESCRIPTION
**Pull request by qcq**
convert spaces to tab for code style
refactor fragment of code to remove one macro

### CODE REVIEW

File: modules/pam_faillock/main.c [hunk 1]
```diff
@@ -129,17 +128,14 @@ do_user(struct options *opts, const char *user)
...
-#endif
-
 		while ((rv=ftruncate(fd, 0)) == -1 && errno == EINTR);
 		if (rv == -1) {
 			fprintf(stderr, "%s: Error clearing the tally file for %s:",
 				opts->progname, user);
 			perror(NULL);
 #ifdef HAVE_LIBAUDIT
 		}
+		int audit_fd;
```
Comment (t8m) [inline modules/pam_faillock/main.c:138]: We do not allow variable declarations in the middle of block. This has to stay as it was.

File: modules/pam_faillock/main.c [hunk 2]
```diff
@@ -201,8 +197,9 @@ do_allusers(struct options *opts)
 	for (i = 0; i < rv; i++) {
 		if (userlist[i]->d_name[0] == '.') {
 			if ((userlist[i]->d_name[1] == '.' && userlist[i]->d_name[2] == '\0') ||
-			    userlist[i]->d_name[1] == '\0')
-				continue;
+				userlist[i]->d_name[1] == '\0') {
+					continue;
```
Comment (t8m) [inline modules/pam_faillock/main.c:201]: I do not really see this as an improvement.

**Review by t8m — CHANGES_REQUESTED** [review]

**Comment by t8m** [comment]
Also the subject line of the commit should contain some meaningful short description. Not just `pam_faillock:`

File: modules/pam_faillock/main.c (hunk 1 above)
Comment (qcq) [reply modules/pam_faillock/main.c:138]: ok, however in my view can remove one macro

File: modules/pam_faillock/main.c (hunk 2 above)
Comment (qcq) [reply modules/pam_faillock/main.c:201]: convert the spaces to tab, also add pair {} to continue statement.

**Review by t8m — APPROVED** [review]
OK now.

File: modules/pam_faillock/main.c [hunk 3]
```diff
@@ -201,8 +200,9 @@ do_allusers(struct options *opts)
...
+				}
```
Comment (ldv-alt) [inline modules/pam_faillock/main.c:205]: I don't mind adding curly brackets if you think it could help, but the new indentation is misleading.

**Review by ldv-alt — CHANGES_REQUESTED** [review]

File: modules/pam_faillock/main.c (hunk 3 above)
Comment (qcq) [reply modules/pam_faillock/main.c:205]: the original indentation is space, just convert that to tab, which can be told by the line 204.

Comment (ldv-alt) [reply modules/pam_faillock/main.c:205]: As you can see, `userlist[i]->d_name[1] == '\0'` is indented under `(userlist[i]->d_name[1] == '.' && userlist[i]->d_name[2] == '\0')`, so this conversion from spaces to tab breaks it.
Also, this conversion changes the indentation of `continue` from 4 tabs to 5 tabs, and indents the closing `}` one tab more that the matching `if` statement.  Both of these changes are misleading.
Could you leave this hunk unchanged, please?

Comment (qcq) [reply modules/pam_faillock/main.c:205]: sure

**Review by t8m — APPROVED** [review]

**Review by ldv-alt — APPROVED** [review]

**Comment by t8m** [comment]
Merged. Thank you for the PR.

**Pull request merged**